src/solvers/dg_simplices/fluxdiff.jl

```diff
@@ -0,0 +1,182 @@
+"""
+  function hadamard_sum_ATr!(du, ATr, volume_flux, u, skip_index=(i,j)->false)
+
+Computes the flux difference ∑_j Aij * f(u_i, u_j) and accumulates the result into `du`. 
+- `du`, `u` are vectors
+- `ATr` is the transpose of the flux differencing matrix `A`. The transpose is used for 
+faster traversal since matrices are column major in Julia. 
+"""
+@inline @muladd function hadamard_sum_ATr!(du, ATr, volume_flux::VF, u, skip_index=(i,j)->false) where {VF}
```

Comment:
Could you please avoid the `@muladd` annotation for specific functions and just wrap the files inside `@muladd begin ... end` blocks` with the same comment as in the rest of Trixi?
```suggestion
@inline function hadamard_sum_ATr!(du, ATr, volume_flux::VF, u, skip_index=(i,j)->false) where {VF}
```

Reply:
Sure, can do.